src/components/TabLinks.tsx

```diff
@@ -14,12 +14,13 @@ type TabLinksProps = {
 
 export const TabLinks: React.FC<TabLinksProps> = ({ tabs }) => {
   return (
-    <div className="flex items-center justify-center pt-6">
+    <div className="mt-3 flex items-center justify-center">
       <div className="flex w-full max-w-md flex-col">
         <div className="flex items-center justify-between">
```

Comment:
複雑になってるんですけどこれで良いのでは...？
```suggestion
    <div className="flex">
```

Reply:
ほんとですね😅